sycl/include/CL/sycl/detail/event_impl.hpp

```diff
@@ -9,8 +9,9 @@
 #pragma once
 
 #include <CL/sycl/detail/common.hpp>
-#include <CL/sycl/detail/event_info.hpp>
 #include <CL/sycl/detail/pi.hpp>
+#include <CL/sycl/detail/plugin.hpp>
```

Comment:
You have both forward-declaration of `plugin` class and corresponding include - is it necessary?